test/k8s-integration/driver.go

```diff
@@ -8,16 +8,18 @@ import (
 	"path/filepath"
 	"time"
 
+	"k8s.io/klog"
 	"k8s.io/kubernetes/test/e2e/framework/podlogs"
 )
 
 func getOverlayDir(pkgDir, deployOverlayName string) string {
 	return filepath.Join(pkgDir, "deploy", "kubernetes", "overlays", deployOverlayName)
 }
 
-func installDriver(goPath, pkgDir, stagingImage, stagingVersion, deployOverlayName string, doDriverBuild bool) error {
+func installDriver(platform, goPath, pkgDir, stagingImage, stagingVersion, deployOverlayName string, doDriverBuild bool) error {
 	if doDriverBuild {
 		// Install kustomize
+		klog.Infof("kustomize")
```

Comment:
Can you make this more descriptive, like "Installing kustomize"

Reply:
updated